### PR TITLE
[pytorch] Add team marker for pytorch tests

### PR DESCRIPTION
**Pull request by tejaschumbalkar**
*GitHub Issue #, if available:*

**Note**: 
- If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

- All PR's are checked weekly for staleness. This PR will be closed if not updated in 30 days.

### Description

Add team markers for pytorch tests

### Tests run

**NOTE: By default, docker builds are disabled. In order to build your container, please update dlc_developer_config.toml and specify the framework to build in "build_frameworks"**
- [ ] I have run builds/tests on commit <INSERT COMMIT ID> for my changes.

**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

### Formatting
- [ ] I have run `black -l 100` on my code (formatting tool: https://black.readthedocs.io/en/stable/getting_started.html)

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/dlc_tests/eks/pytorch/training/test_eks_pytorch_training.py [hunk 1]
```diff
@@ -161,6 +163,7 @@ def test_eks_pt_s3_plugin_single_node_training(pytorch_training, outside_version
 @pytest.mark.skip_dgl_test
 @pytest.mark.integration("dgl")
 @pytest.mark.model("gcn")
+@pytest.mark.team("conda")
```
Comment (roywei) [inline test/dlc_tests/eks/pytorch/training/test_eks_pytorch_training.py:166]: should this go to DGL team?

File: test/sagemaker_tests/pytorch/training/integration/sagemaker/test_dgl.py [hunk 2]
```diff
@@ -37,6 +37,7 @@
 @pytest.mark.model("gcn")
 @pytest.mark.skip_gpu
 @pytest.mark.skip_py2_containers
+@pytest.mark.team("conda")
```
Comment (roywei) [inline test/sagemaker_tests/pytorch/training/integration/sagemaker/test_dgl.py:40]: DGL team

**Review by roywei — CHANGES_REQUESTED** [review]
Thanks for going through these tests, let's redirect inductor, graviton and dgl tests, rest LGTM

File: test/dlc_tests/benchmark/ec2/pytorch/inference/test_performance_pytorch_inference.py [hunk 3]
```diff
@@ -60,6 +62,7 @@ def test_performance_ec2_pytorch_inference_cpu(pytorch_inference, ec2_connection
 @pytest.mark.model("resnet18, VGG13, MobileNetV2, GoogleNet, DenseNet121, InceptionV3")
 @pytest.mark.parametrize("ec2_instance_type", ["c6g.4xlarge"], indirect=True)
 @pytest.mark.parametrize("ec2_instance_ami", [UL20_CPU_ARM64_US_WEST_2], indirect=True)
+@pytest.mark.team("conda")
```
[the remark this answers is not in the document]
Comment (tejaschumbalkar) [reply test/dlc_tests/benchmark/ec2/pytorch/inference/test_performance_pytorch_inference.py]: addressed

File: test/dlc_tests/benchmark/ec2/pytorch/training/test_performance_pytorch_training.py [hunk 4]
```diff
@@ -90,6 +92,7 @@ def test_performance_pytorch_gpu_imagenet(pytorch_training, ec2_connection, gpu_
 )
 @pytest.mark.parametrize("ec2_instance_type", [PT_EC2_GPU_INDUCTOR_INSTANCE_TYPES], indirect=True)
 @pytest.mark.skip  # skipping inductor benchmarking test for now due to incomplete implementation
+@pytest.mark.team("conda")
```
[the remark this answers is not in the document]
Comment (tejaschumbalkar) [reply test/dlc_tests/benchmark/ec2/pytorch/training/test_performance_pytorch_training.py]: addressed

File: test/dlc_tests/benchmark/sagemaker/pytorch/training/test_performance_inductor.py [hunk 5]
```diff
@@ -45,6 +45,7 @@ def sagemaker_session(region):
 @pytest.mark.model("huggingface")
 @pytest.mark.parametrize("instance_type", instance_types, indirect=True)
 @pytest.mark.skip  # skip for now due to pending design decision
+@pytest.mark.team("conda")
```
[the remark this answers is not in the document]
Comment (tejaschumbalkar) [reply test/dlc_tests/benchmark/sagemaker/pytorch/training/test_performance_inductor.py]: addressed

File: test/dlc_tests/eks/pytorch/training/test_eks_pytorch_training.py (hunk 1 above)
Comment (tejaschumbalkar) [reply test/dlc_tests/eks/pytorch/training/test_eks_pytorch_training.py]: addressed

File: test/sagemaker_tests/pytorch/training/integration/sagemaker/test_dgl.py (hunk 2 above)
Comment (tejaschumbalkar) [reply test/sagemaker_tests/pytorch/training/integration/sagemaker/test_dgl.py]: addressed

**Comment by tejaschumbalkar** [comment]
- Removed the ownership for graviton, dgl and inductor test.
- Changed the github team in codeowners

**Review by junpuf — CHANGES_REQUESTED** [review]
Found some more inductor tests to be re-linked

File: test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py [hunk 6]
```diff
@@ -337,6 +348,7 @@ def test_pytorch_nccl(pytorch_training, ec2_connection, gpu_only, py3_only, ec2_
 @pytest.mark.model("resnet18")
 @pytest.mark.parametrize("ec2_instance_type", PT_INDUCTOR_TEST_INSTANCE_TYPE, indirect=True)
 @pytest.mark.skip_inductor_test
+@pytest.mark.team("conda")
 def test_pytorch_nccl_inductor(
```
[the remark this answers is not in the document]
Comment (tejaschumbalkar) [reply test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py:352]: done

File: test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist_inductor.py [hunk 7]
```diff
@@ -45,6 +45,7 @@
 @pytest.mark.model("mnist")
 @pytest.mark.processor("cpu")
 @pytest.mark.cpu_test
+@pytest.mark.team("conda")
```
[the remark this answers is not in the document]
Comment (tejaschumbalkar) [reply test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist_inductor.py]: done

**Review by roywei — APPROVED** [review]
LGTM!

**Review by junpuf — APPROVED** [review]
LGTM

**Pull request merged**